console_conf/controllers/identity.py

```diff
@@ -31,9 +31,39 @@ def default(self):
         self.ui.set_header(title, excerpt)
         self.ui.set_footer(footer, 40)
         self.ui.set_body(self.identity_view(self.model, self.signal, self.opts, self.loop))
+        device_owner = self.get_device_owner()
+        if device_owner is not None:
+            # FIXME: should be email rather than realname here, but extrausers
+            # only has an email in realname.
+            self.signal.emit_signal('identity:done', device_owner['realname'])
```

Comment:
I still think calling self.model.add_user() here and then directly going to the 'identity:login' makes more sense but I'm not going to make a big fuss about it.